db/migrate/20150604154825_rename_container_node_conditions_to_container_conditions.rb

```diff
@@ -0,0 +1,26 @@
+class RenameContainerNodeConditionsToContainerConditions < ActiveRecord::Migration
```

Comment:
A migration spec (see https://github.com/ManageIQ/manageiq/tree/master/spec/migrations) would be helpful.